user-service/controller/user-controller.ts

```diff
@@ -132,6 +135,12 @@ export const deleteUser: RequestHandler = async (req, res, next) => {
 
   await _deleteUser(user);
   console.log(`-- User ${username} Successfully Deleted --`);
+  const historyServiceURI = process.env.HISTORY_SERVICE_TEST!;
```

Comment:
tho this increasing some coupling, i guess sometimes is a give and take of microservices